README.md

```diff
@@ -90,13 +90,36 @@ greatly improving installation time and effort, especially under windows.
 - download the source distribution from our github repository:
     https://github.com/EMS-TU-Ilmenau/fastmat/archive/master.zip
 - unpack its contents and navigate to the project root directory
-- make sure you have installed all listed dependencies with at least the
-  specified version
-- type `make install` to install fastmat on your computer
-    * If you intend to install the package locally for your user
-      type `make install MODE=--user` instead.
-    * If you only would like to compile the package to use it from this local
-      directory without installing it, type `make compile`
+#### LINUX
```

Comment:
These sections should be cleaned up. As this is a very very old description, it does not feature pip at all, which is currently the preferred isntallation method. I would suggest restructuring the whole installation chapter to the following key points:

# Intallation

## using pip
 --> pip install fastmat

## manually
  put descriptions on where to get the source from, what to do with it all those descriptions on make file usage here (also point to the relevant wiki page of this repo)

## Requirements for Building on Windows
  as no compiler exists natively ... bla bla
  our recommendations: Intel Python, ... bla bla